examples/deadline/All-In-AWS-Infrastructure-SEP/python/package/lib/sep_stack.py

```diff
@@ -34,6 +57,10 @@ class SEPStackProps(StackProps):
     """
     # The path to the directory where the staged Deadline Docker recipes are.
     docker_recipes_stage_path: str
+    # The IMachineImage to use for Workers (needs Deadline Client installed).
+    worker_machine_image: IMachineImage
+    # The name of the EC2 keypair to associate with Worker nodes.
+    key_pair_name: Optional[str]
 
 
 class SEPStack(Stack):
```

Comment:
This is a good point. Although, I don't want to change it at the moment, as it will increase the scope of this already big PR. This seems like a good follow-up task. Maybe we could do it later together with adding integration tests.